### ISSUE TITLE
next-tinacms-json writes a `default` object to source file

### ISSUE THREAD

**Issue by kendallstrautman** [tinacms/tinacms#676]
## Description

When getting JSON data with Next's `getInitialProps`, the return object has a `default` property that seems just like a duplicate of the source data. When using helpers from `next-tinacms-json`, this `default` data is written to the source file. The problem is that when the data changes, the old `default` JSON objects aren't deleted, so we start to amass this collection of outdated `default` JSON objects. 

### Steps to reproduce

Start up `demo-next` in the repo, make some edits to the content and look at the source file. You can see the results of this behavior already in [index.json](https://github.com/tinacms/tinacms/blob/master/packages/demo-next/data/index.json).

### Expected result

Only the original properties from the source file are written in the first place. And even if the `default` object was also written, it would be updated or the old `default` objects would be overwritten.

### Actual result

The `default` object is written to the source file, and when the content changes and the page is refreshed, a new `default` object is added. 

### Suggestions

We could remove this default object within the helpers, but I need to have a better understanding of its function in next.js

#### Tinacms Package Versions

@tinacms/api-git@0.5.3
@tinacms/git-client@0.4.1
next-tinacms-json@0.1.2
next-tinacms-markdown@0.1.0-canary.2441
 tinacms@^0.12.0-alpha.0

Next.js & `next-tinacms-json`

#### Tools

| Tool    | Version |
| ------- | ------- |
| browser |    Firefox     |
| os      |    Mac     |
| node    |    13.5.0     |
| npm     |   6.13.4      |

**Comment by alexbarbato** [comment]
I tried poking at this some this weekend. I couldn't figure out why exclusively importing the json file in getInitialProps would add that default key, as importing the module statically doesn't cause the same problem.

Happy to keep poking and interested if someone else knows why!

**Comment by ncphillips** [comment]
Since Javascript modules have default exports, my assumption is this is not a bug in `next-tinacms-json` but actually a quirk of= `webpack`.

The [markdown](https://github.com/tinacms/tinacms/blob/master/packages/demo-next/pages/info.js#L68-L69) example pulls the content from `default` so this is probably how it should be done for json as well.

I'm guessing that webpack does some weird thing where it just puts all the json files props as named exports, but also leaves the entire object in the `default` export. In that case I would load the [configData](https://github.com/tinacms/tinacms/blob/master/packages/demo-next/pages/index.js#L58-L59) like so:

```
const configData = await import('../data/config.json').default
```

**Comment by kendallstrautman** [comment]
In testing it out, this approach works:

```js 
  const indexData = await import(`../data/index.json`)

  return {
    jsonFile: {
      fileRelativePath: `data/index.json`,
      data: indexData.default,
    },
  }
```

**Comment by kendallstrautman** [comment]
I can add a note about this in the docs

**Comment by ncphillips** [comment]
> indexData seems to return undefined when I try to access default on import

That makes sense. It would actually need to be:

```tsx
const indexData = (await import(`../data/index.json`)).default
```

Otherwise it would be

```tsx
const promiseForIndexData = import(`../data/index.json`)

console.log(promiseForIndexData.default) // `undefined`

const indexData = await promiseForIndexData

console.log(indexData) // { ... } 
```

**Comment by kendallstrautman** [comment]
[Docs](https://github.com/tinacms/tinacms-site/pull/232) and [blog](https://github.com/tinacms/tinacms-site/pull/236) examples updated.

**Comment by marklawlor** [comment]
Has this issue regressed? I'm currently seeing the same behaviour with these versions

    "@tinacms/api-git": "^0.7.0",
    "@tinacms/git-client": "^0.6.0",
    "next": "^9.3.0",
    "next-tinacms-json": "^0.3.5",
    "tinacms": "^0.16.0"
    "typescript": "^3.8.3"

A possible difference is I'm using Typescript

**Comment by kendallstrautman** [comment]
Hi @marklawlor — are you passing the [default object](https://github.com/tinacms/tinacms-site/pull/236/commits/eb192542329fd250d5363b919a575fed2190e44e) in the return value?

**Comment by marklawlor** [comment]
I believe this is correct

```
Page.getInitialProps = async function() {
  const indexData = await import(`../data/index.json`);
  return {
    jsonFile: {
      fileRelativePath: `src/data/index.json`,
      data: indexData.default
    }
  };
};
```

**Comment by kendallstrautman** [comment]
Hmm....yeah that looks right to me.

**Comment by marklawlor** [comment]
Appears fixed in "tinacms": "^0.17.0"
